domainobjs/ts/publicKey.ts

```diff
@@ -0,0 +1,155 @@
+import * as assert from "assert";
+import {
+    SNARK_FIELD_SIZE,
+    hashLeftRight,
+    packPubKey,
+    unpackPubKey,
+    PubKey as RawPubKey,
+} from "maci-crypto";
+import { SERIALIZED_PUB_KEY_PREFIX } from "./constants";
+
+/**
+ * @notice A class representing a public key
+ * @note This is a MACI public key, which is not to be
```

Comment:
`@note` vs `@notice`?

Reply:
Overall I like this explanation though (that MACI key != EVM key)! Would it make sense to add this comment to the `PrivKey` class as well?

Reply:
I think worth adding these explanations to the README as well (e.g. https://github.com/privacy-scaling-explorations/maci/tree/dev/domainobjs#pubkey)

Reply:
Adding to the other classes too and README